doc/news/changes/minor/20190313LucaHeltai

```diff
@@ -0,0 +1,6 @@
+New: Added two new classes to the MeshWorker namespace: MeshWorker::ScratchData and 
+MeshWorker::CopyData, that can be used as good default classes with the 
+WorkStream::run and MeshWorker::mesh_loop functions.
```

Comment:
`WorkStream::run()`
`MeshWorker::mesh_loop()`

I see on http://www.doxygen.nl/manual/autolink.html#linkfunc that what you've written here should generate a link. But from https://dealii.org/developer/developers/writing-documentation.html it seems to me that do prefer to have the function braces. Feel free to ignore if I'm wrong.